internal/clients/composer/openapi.v2.yml

```diff
@@ -1764,6 +1764,12 @@ components:
         key:
           type: string
           example: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAINrGKErMYi+MMUwuHaRAJmRLoIzRf2qD2dD5z0BTx/6x"
+        password:
+          type: string
+          description: |
+            If the password starts with $6$, $5$, or $2b$ it will be stored as
+            an encrypted password. Otherwise it will be treated as a plain text
+            password.
```

Comment:
Composer uses Anaconda to do the installation which supports both plain text and hashed passwords.

The code I am seeing here does perform hashing of plain text passwords. Which is I think a good idea since blueprint is stored and being dragged among several services. Hashed passwords are safe to store.